src/EA.Weee.RequestHandlers/Mappings/EvidenceNoteDataMap.cs

```diff
@@ -24,12 +24,12 @@ public ListOfEvidenceNoteDataMap Map(ListOfNotesMap source)
             if (source.ListOfNotes.Any())
             {
                 foreach (var note in source.ListOfNotes)
-                { 
+                {
                     var evidenceNoteData = mapper.Map<EvidenceNoteWithCriteriaMap, EvidenceNoteData>(new EvidenceNoteWithCriteriaMap(note)
                     {
                         CategoryFilter = source.CategoryFilter,
                         IncludeTonnage = source.IncludeTonnage,
-                        IncludeHistory = false
+                        IncludeHistory = note.Status == Domain.Evidence.NoteStatus.Void
```

Comment:
Set to false here. We dont want to include history when this is the notes to be displayed in the tables.

Reply:
done